wordpress-mobile/WordPress-iOS
# ULS: update Auth to show email and gravatar on Password view

Ref: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/issues/352
Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/359

To test:

---
- Enable the `unifiedGoogle` feature flag.
- Select `Continue with Google` from either login or signup.
- Select a Google account that:
  - Is disconnected from WP.com. WP.com > Me > Security > Social Logins > Google > disconnect
  - Has a WP password set.
  - Does not have a gravatar.
- On the Password view, verify the placeholder icon and the email is displayed:

<kbd>![no_gravatar](https://user-images.githubusercontent.com/1816888/89355413-90ae8f00-d678-11ea-8dae-1e6e1c2320a7.png)</kbd>

---
- Select a Google account that:
  - Is disconnected from WP.com. WP.com > Me > Security > Social Logins > Google > disconnect
  - Has a WP password set.
  - Does have a gravatar.
- On the Password view, verify the gravatar and the email are displayed:

<kbd>![gravatar](https://user-images.githubusercontent.com/1816888/89355480-b5a30200-d678-11ea-9e0a-d0e1ff166e08.png)</kbd>

---
PR submission checklist:
